src/library.js

```diff
@@ -4284,12 +4284,6 @@ LibraryManager.library = {
   // USE_FULL_LIBRARY hacks
   realloc: function() { throw 'bad' },
 
-  // internal musl requirements that we do, for now
-  _pthread_cleanup_push: function(){},
-  _pthread_cleanup_pop: function(){},
-  __pthread_self: function() { abort() },
```

Comment:
The functions `__pthread_self` and `__pthread_setcancelstate` seem to be some kind of leftovers, they are not needed at all actually (either in pthreads enabled builds or otherwise)